source/_components/plant.markdown

```diff
@@ -45,8 +46,10 @@ Configuration variables:
   - **max_conductivity** (*Optional*): Maximum conductivity level before triggering a problem. Typical value: 3000
   - **min_temperature** (*Optional*): Minimum temperature before triggering a problem.
   - **max_temperature** (*Optional*): Maximum temperature before triggering a problem.
-  - **min_brightness** (*Optional*): Minimum brightness before triggering a problem.
+  - **min_brightness** (*Optional*): Minimum brightness before triggering a problem. In contrast to the other values this check is *not* looking at the current situation, but rather at the last days. A problem is only reported if the maximum brightness over the last days was lower than min_brightness. You can use this to check if the plant gets enough light during the course of the day.
```

Comment:
**Spelling & Grammar**:
- Missing comma after `values` -> `the other values, this check is not`

Reply:
fixed